src/interfaces.ts

```diff
@@ -20,77 +17,60 @@ export interface ConnectionOptions {
   signer?: Signer;
   host?: string;
   network?: string;
+  contract?: string;
 }
 
 export interface RpcParams {
-  controller?: boolean;
-  createStatement?: string;
-  description?: string;
-  dryrun?: boolean;
-  statement?: string;
-  tableId?: string;
-}
-
-export interface RpcRequestParam {
   controller?: string;
   /* eslint-disable-next-line camelcase */
   create_statement?: string;
-  description?: string;
-  dryrun?: boolean;
-  id?: string;
   statement?: string;
-}
-
-export interface ConnectionReceipt {
-  jwsToken: Token;
-  ethAccounts: Array<string>;
+  /* eslint-disable-next-line camelcase */
+  txn_hash?: string;
 }
 
 export interface SupportedNetwork {
   key: string;
+  name: string; // Matches naming convention used in https://chainlist.org and ethers network lib
   phrase: string;
-}
-
-/**
- * ColumnDescriptor gives metadata about a colum (name, type)
- */
-export interface ColumnDescriptor {
-  name: string;
+  chainId: number;
 }
 
 export type KeyVal<T = any> = [string, T];
 
-export type Column = Array<ColumnDescriptor>;
-
-export type Row = Array<string | number | boolean>;
+export type Columns = Array<{ name: string }>;
```

Comment:
Fixing this to match actual response. Looks like there was a "double" wrapping in an Array.